src/java/test/org/apache/zookeeper/client/StaticHostProviderTest.java

```diff
@@ -110,6 +110,71 @@ public void testLiteralIPNoReverseNS() throws Exception {
         }
     }
 
+    @Test
+    public void testReresolutionAfterNoConnection() throws Exception {
+        byte size = 1;
+        ArrayList<InetSocketAddress> addresses = new ArrayList<InetSocketAddress>(size);
```

Comment:
Why specify the size here at all?

Reply:
Dunno, lifted straight from the upstream tests.

Reply:
It does seem a bit odd.